### PR TITLE
HOCS-3287 Check your answers screen

### PR DESCRIPTION
**Pull request by agraham19**
HOCS-3287 Check your answers screen
    * Add check your answers screen to creation stage

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package uk.gov.digital.ho.hocs.workflow.api.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java:8]: @agraham19 I think these are bits of my branch that have made it in here. I've taken them out on the branch we're merging too. `@ToString` is great when debugging, there's always the risk that something will appear in logs that shouldn't and our logging doesn't really require objects to be logged out in full. That being said, if you're not familiar it, I'd recommend learning it as it is easily configured to remove the things you don't want to log.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java [hunk 2]
```diff
@@ -21,6 +24,7 @@
     private LocalDate dateReceived;
 
     @JsonProperty("data")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java:27]: Again i think this is a residual left from my branch. It's not necessary as the front end always sends back at least `data = {}`.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsFormField.java [hunk 3]
```diff
@@ -13,6 +14,7 @@
 @NoArgsConstructor
 @AllArgsConstructor()
 @Getter
+@ToString
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsFormField.java:17]: as above

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsForm.java [hunk 4]
```diff
@@ -3,10 +3,12 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 
 import java.util.Map;
 
 @AllArgsConstructor
+@ToString
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsForm.java:11]: as above

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsCaseSchema.java [hunk 5]
```diff
@@ -3,12 +3,14 @@
...
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.ToString;
 
 import java.util.List;
 import java.util.Map;
 
 @AllArgsConstructor
 @Getter
+@ToString
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsCaseSchema.java:13]: as above

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetStageResponse.java [hunk 6]
```diff
@@ -3,12 +3,14 @@
...
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.ToString;
 import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsForm;
 
 import java.util.UUID;
 
 @AllArgsConstructor
 @Getter
+@ToString
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetStageResponse.java:13]: as above

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetCaseResponse.java [hunk 7]
```diff
@@ -4,11 +4,13 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.Setter;
+import lombok.ToString;
 import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsCaseSchema;
 import java.util.Map;
 
 @AllArgsConstructor
 @Getter
+@ToString
```
Comment (gary-newbolt) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetCaseResponse.java:13]: as above

**Review by gary-newbolt — COMMENTED** [review]
Just a little tidy up... I'm not sure how some have crept through. But looks good.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java (hunk 1 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java (hunk 2 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsForm.java (hunk 4 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsForm.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsFormField.java (hunk 3 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsFormField.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsCaseSchema.java (hunk 5 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsCaseSchema.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetStageResponse.java (hunk 6 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetStageResponse.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetCaseResponse.java (hunk 7 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/GetCaseResponse.java]: I have addressed this.

**Review by gary-newbolt — APPROVED** [review]
lgtm

**Pull request merged**